chore(curriculum): add colored markers to new cert (#56718)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-colored-markers/616d3a67ccf800ad94ec89ae.md

---
id: 616d3a67ccf800ad94ec89ae
title: Step 4
challengeType: 0
dashedName: step-4
---

# --description--

> Now you'll add some elements that you'll eventually style into color markers.

Reviewer: ```suggestion
Now, you'll add some elements that you'll eventually style as color markers.
```

> First, within the `body` element, add a `div` element and set its `class` attribute to `container`. Make sure the `div` element is below the `h1` element.

Reviewer: ```suggestion
First, within the `body` element, add a `div` element with a `class` attribute set to `container`. Make sure the `div` element is placed below the `h1` element.
```
